packages/remix-react/components.tsx

```diff
@@ -764,6 +796,7 @@ import(${JSON.stringify(manifest.entry.module)});`;
         />
         <script
           {...props}
+          suppressHydrationWarning
```

Comment:
Any reason this wasn't there already in dev?  We added it while debugging some other integration test hydration failures, which I think came down to the lack of a `<head>` in our `root.tsx` but this felt like a good change?

Reply:
Leave this in 👍